scikeras/_types.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable, List, Type, Union
+
+import numpy as np
+import tensorflow as tf
+import tensorflow.keras as keras
+
+from tensorflow.keras.callbacks import Callback as TF_Callback
+from tensorflow.keras.losses import Loss as TF_Loss
+from tensorflow.keras.metrics import Metric as TF_Metric
+from tensorflow.keras.optimizers import Optimizer as TF_Optimizer
+
+
+Model = Union[Callable[..., keras.Model], keras.Model]
+RandomState = Union[int, np.random.RandomState]
+Optimizer = Union[str, TF_Optimizer, Type[TF_Optimizer]]
+Loss = Union[str, TF_Loss, Type[TF_Loss], Callable]
+Metrics = Union[List[Union[str, TF_Metric, Type[TF_Metric], Callable]]]
+Callbacks = Union[List[Union[TF_Callback, Type[TF_Callback]]]]
```

Comment:
I think this file should be renamed to ` py.typed` for future MyPy compatibility, etc. But not 100% sure on that.

Reply:
It looks like the file only needs to be created: https://blog.whtsky.me/tech/2021/dont-forget-py.typed-for-your-typed-python-package/

Reply:
...but I'm not sure how to instruct PyPI/etc how to install that file in pyproject.toml.

Reply:
Thank for checking. It looks like [this](https://github.com/python-poetry/poetry/issues/1338#issuecomment-703737178) is how. But don't worry about it, I can take care of that in a separate PR.